internal/pkg/skuba/gangway/gangway.go

```diff
@@ -91,6 +94,12 @@ func GetClientSecret(client clientset.Interface) string {
 		return gClientSecret
 	}
 
+	// generate a new client secret when `skuba cluster init`
+	if client == nil {
```

Comment:
will `getClientSecretFromConfigMap` not return an error if `client` is nil? if it does, this is not needed and seems to just be extra logic?

Reply:
An exception happens when executes
`client.CoreV1().ConfigMaps(metav1.NamespaceSystem).Get(configmapName, metav1.GetOptions{})`.